ui/pages/confirmations/components/confirm/info/shared/gas-fees-details/gas-fees-details.tsx

```diff
@@ -101,7 +101,7 @@ export const GasFeesDetails = ({
           </Box>
         </ConfirmInfoAlertRow>
       )}
-      {showAdvancedDetails && (
+      {showAdvancedDetails && !transactionMeta.selectedGasFeeToken && (
```

Comment:
Nit: will be nice to unit test cover this, but may be in a following PR.

Reply:
Agreed, I've done more extensive unit tests for all the new components, but most of the existing ones were just a single snapshot test. Happy to follow up in another PR.